<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<qualifiedTypeNameHistroy>
    <fullyQualifiedTypeName name="java.net.ConnectException"/>
    <fullyQualifiedTypeName name="com.mysql.cj.xdevapi.Statement"/>
    <fullyQualifiedTypeName name="java.sql.Statement"/>
    <fullyQualifiedTypeName name="java.util.Scanner"/>
    <fullyQualifiedTypeName name="java.lang.System"/>
    <fullyQualifiedTypeName name="org.eclipse.jdt.internal.compiler.flow.ExceptionHandlingFlowContext"/>
    <fullyQualifiedTypeName name="java.lang.Class"/>
    <fullyQualifiedTypeName name="java.util.List"/>
    <fullyQualifiedTypeName name="java.sql.Connection"/>
    <fullyQualifiedTypeName name="java.sql.PreparedStatement"/>
    <fullyQualifiedTypeName name="java.sql.ResultSet"/>
    <fullyQualifiedTypeName name="java.util.ArrayList"/>
    <fullyQualifiedTypeName name="kr.co.ict.UserVO"/>
    <fullyQualifiedTypeName name="java.sql.DriverManager"/>
    <fullyQualifiedTypeName name="java.sql.SQLException"/>
    <fullyQualifiedTypeName name="java.lang.Exception"/>
    <fullyQualifiedTypeName name="kr.co.ict.UserDAO"/>
    <fullyQualifiedTypeName name="javax.naming.Context"/>
    <fullyQualifiedTypeName name="javax.naming.InitialContext"/>
    <fullyQualifiedTypeName name="javax.sql.DataSource"/>
</qualifiedTypeNameHistroy>
